pkg/engine/jmespath/functions_test.go

```diff
@@ -972,3 +972,42 @@ func Test_PathCanonicalize(t *testing.T) {
 		})
 	}
 }
+
+func Test_Truncate(t *testing.T) {
+	// Can't use integer literals due to
+	// https://github.com/jmespath/go-jmespath/issues/27
+	//
+	// TODO: fix this in https://github.com/kyverno/go-jmespath
```

Comment:
Do we want to track it via an issue?

Reply:
This is the same comment as in Replace that also has a Number as an argument (I just copied it over here)